### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Fraser999**
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/safe_vault/375)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @maqi

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/personas/maid_manager.rs [hunk 1]
```diff
@@ -487,7 +495,8 @@ mod test {
         // put_requests[0] - account creation.
         assert_eq!(put_requests.len(), 2);
         assert_eq!(put_requests[1].src, env.our_authority);
-        assert_eq!(put_requests[1].dst, Authority::NaeManager(immutable_data.name()));
```
Comment (maqi) [inline src/personas/maid_manager.rs]: is this complained by rustfmt or clippy?
I think it is definitely less than 100 characters.
does it meant we are switching to 80 characters rule,
or just for assert_eq macro, it must be below 80 characters?

Comment (Fraser999) [inline src/personas/maid_manager.rs]: It's rustfmt.  It seems to have a random approach to handling macros.

**Comment by maqi** [comment]
Reviewed 2 of 5 files at r1, 3 of 3 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/safe_vault/375)*

**Pull request merged**